lua/fidget/integration/xcodebuild-nvim.lua

```diff
@@ -0,0 +1,86 @@
+local M = {}
+
+-- Only register one callback, so setup() can be called multiple times.
+local already_subscribed = false
+
+---@options integration.xcodebuild-nvim [[
+--- xcodebuild.nvim integration
+M.options = {
+  --- Integrate with wojciech-kulik/xcodebuild.nvim (if installed)
+  ---
+  --- Dynamically offset Fidget's notifications window when the Test Explorer window
+  --- is open on the right side + the Fidget window is "editor"-relative.
+  ---
+  ---@type boolean
+  enable = true,
+}
+---@options ]]
+
+require("fidget.options").declare(
+  M,
+  "integration.xcodebuild-nvim",
+  M.options,
+  function()
+    if not M.options.enable or already_subscribed then
+      return
+    end
+
+    local ok, _ = pcall(require, "xcodebuild")
+    if not ok then
+      return
+    end
+
+    local win = require("fidget.notification.window")
+    local test_explorer_winid = nil
+
+    local function resize(winid)
+      test_explorer_winid = winid
+
+      if win.options.relative == "editor" then
+        local col = vim.api.nvim_win_get_position(winid)[2]
+
+        if col > 1 then
+          local width = vim.api.nvim_win_get_width(winid)
+          win.set_x_offset(width + 1)
```

Comment:
Same question as before, about adjusting the `x_offset` in the nvim-tree integration